### PR TITLE
chore: optimize c003 test and use `expect`

### PR DESCRIPTION
**Pull request by Rqnsom**
~Two commits:~
EDIT:
Three commits:

```git
chore: optimize c003 test

- include extra comments for the rpc.rs file
```

```
chore: avoid using unwrap

- rather expect everything :^)
```
```
chore: align crash message styles

- expect/panic/assert messages should all be lower-case messages
```

### CODE REVIEW

File: src/tests/conformance/post_handshake/query/get_block.rs [hunk 1]
```diff
@@ -33,37 +33,30 @@ async fn c003_V1_BLOCK_ROUND_get_block() {
...
+                block_cert.block.prevous_block_hash.is_none(),
+                "Previous block hash shouldn't be found for the first round"
+            );
+        } else {
+            assert!(
+                block_cert.block.prevous_block_hash.is_some(),
+                "Previous block hash not found"
+            );
+        }
+    }
```
Comment (Rqnsom) [inline src/tests/conformance/post_handshake/query/get_block.rs:59]: I would have used closure here, but an async block is not allowed in a non-async closure, and those are still unstable, unfortunately.

File: src/tests/conformance/handshake.rs [hunk 2]
```diff
@@ -23,18 +23,23 @@ async fn c001_handshake_when_node_receives_connection() {
         .await
         .expect("Unable to build a synthetic node");
 
+    let net_addr = node.net_addr().expect("Network address not found");
```
Comment (niklaslong) [inline src/tests/conformance/handshake.rs:26]: I believe the convention is to have these be lower case?

**Review by ljedrz — APPROVED** [review]
LGTM :+1:

**Review by niklaslong — APPROVED** [review]
Other than the one comment, lgtm!

File: src/tests/conformance/handshake.rs (hunk 2 above)
Comment (ljedrz) [reply src/tests/conformance/handshake.rs:26]: I usually don't pay much attention to capitalization in panic conditions, but yeah, consistency won't hurt :ok_hand:

Comment (Rqnsom) [reply src/tests/conformance/handshake.rs:26]: > I believe the [convention](https://doc.rust-lang.org/std/error/index.html#common-message-styles) is to have these be lower case?

Yes! I will append one more commit here in this PR to fix this across the repository. Thanks 🙂

Comment (Rqnsom) [reply src/tests/conformance/handshake.rs]: Aaand done

**Review by ljedrz — APPROVED** [review]

**Pull request merged**